manifests/s/SuperFlexibleSoftware/SzArchiver/0.84/SuperFlexibleSoftware.SzArchiver.locale.en-US.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.5.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: SuperFlexibleSoftware.SzArchiver
+PackageVersion: 0.84
+PackageLocale: en-US
+Publisher: Super Flexible Software GmbH & Co. KG
+PackageName: SzArchiver
```

```suggestion
PackageName: SzArchiver
PackageUrl: https://www.syncovery.com/szarchiver/
```
Hi @chausner,

This `PackageUrl` helps me to validate that the InstallerUrl is valid, and will let interested users know where to learn more about this package.